modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -40,3 +41,89 @@ Select one of the configuration options below:
 <1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the PersistentVolume is created.
 
 <2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a StorageClass with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+--
+
+.  Create the following resource objects that are required to deploy an NFS server for metering. Use the `oc create -f <file-name>.yaml` command to create the object YAML files.
+..  Configure a PersistentVolume resource object. For example, `nfs_persistentvolume.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+  labels:
+    role: nfs-server
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteMany
+  storageClassName: nfs-server
+  nfs:
+    path: "/"
+    server: REPLACEME
+  persistentVolumeReclaimPolicy: Delete
+----
+..  Configure a Pod resource object with the `nfs-server` role. For example, `nfs_server.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-server
+  labels:
+    role: nfs-server
+spec:
+  containers:
+    - name: nfs-server
+      image: quay.io/openshifttest/nfs-server
+      imagePullPolicy: IfNotPresent
+      ports:
+        - name: nfs
+          containerPort: 2049
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - mountPath: "/mnt/data"
+        name: local
+  volumes:
+    - name: local
+      emptyDir: {}
+----
+..  Configure a Service resource object with the `nfs-server` role. For example, `nfs_service.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: nfs-service
+  labels:
+    role: nfs-server
+spec:
+  ports:
+  - name: 2049-tcp
+    port: 2049
+    protocol: TCP
+    targetPort: 2049
+  selector:
+    role: nfs-server
+  sessionAffinity: None
+  type: ClusterIP
+----
+..  Configure a StorageClass resource object. For example, `nfs_storageclass.yaml`:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nfs-server
```

Comment:
It might be helpful to let reader know that `metadata.name` here needs to be an exact match with `[kind: PersistentVolume].spec.storageClassName`